src/flashbang/core/FlashbangApp.ts

```diff
@@ -183,7 +189,10 @@ export default class FlashbangApp {
     protected onContextMenuEvent(e: Event): void {
         const {topMode} = this._modeStack;
         if (topMode != null) {
-            topMode.onContextMenuEvent(e);
+            if (Mol3DGate.scope?.isOver3DCanvas) {
+                e.preventDefault();
+                e.stopPropagation();
+            } else topMode.onContextMenuEvent(e);
```

Comment:
As mentioned elsewhere, I think we can just let it display the normal context menu

Reply:
I removed this here and wrote it again in GameMode.ts. This is needed to prevent the context menu from appearing when MRB is released in 3D window after rotation.

Reply:
Ah! Didn’t realize the right button was used for that.